merkletree/proof.go

```diff
@@ -59,6 +59,10 @@ func (ap *AuthenticationPath) authPathHash() []byte {
 	return hash
 }
 
+func (ap *AuthenticationPath) VerifyBinding(key []byte) bool {
+	return bytes.Equal(ap.Leaf.Value, key)
```

Comment:
> I  think it depends on how we store the key and the policy into the binding. That should be addressed in #46.

You're right. I was thinking we should just work without policies now, and only introduce them after we implement key changes.

> Thus, the client has to know the storage schema.

Right yeah, this we decided this in order to give the users some privacy from the server.

> Another question is do we expect this?

I think we definitely want to include the policy flags in the `Value` so the server also commits to these flags. It's the easiest way to ensure users have security against the server changing these flags arbitrarily (i.e. they can be verified alongside keys etc).